content/en/docs/best-practices/channel-naming.md

```diff
@@ -136,21 +146,34 @@ can experiment with that fast version, but the stable version for example
 
 #### Example 2
 
-A possible but less typical case might be where an operator wants to be 
-supported at various operator major/minor versions  For example you might 
-have an operator version at 1.3 and also at 2.4 that you need or want to 
-offer support for at the same time.  However, you might not want to have 
-OLM upgrade users to the 2.4 operator but instead keep them upgrading within 
-the 1.3 versions.   In that case, you would end up with channels as 
+By following the recommendation, you would provide the major versions of the Operators
+in the channel naming. So that your consumers can be subscribed to a channel where
+they can safely install the latest versions published on it and avoid workflow
+issues on their cluster because they know that none of the solutions which will be
+distributed within contains breaking changes:
+
+| Channels for Operator version(s) 2.x | 
+| :------------- | 
+| candidate-2.x |
+| fast-2.x | 
+| stable-2.x |
```

Comment:
I think I'd leave off the `.x` suffix to align with major/minor channel names and make it more clear that these channels track all releases within a major version.